src/dsp_tools/commands/excel2json/input_error.py

```diff
@@ -126,3 +126,39 @@ def execute_error_protocol(self) -> str:
         if self.message_path:
             msg.append(f"The error occurred at {self.message_path}")
         return separator.join(msg)
+
+
+@dataclass(frozen=True)
+class JsonValidationResourceProblem:
+    """This class contains information about a JSON resource section that fails its validation against the schema."""
+
+    problematic_resource: str | None = None
+    excel_sheet: str | None = None
+    excel_column: str | None = None
+    excel_row: int | None = None
```

Comment:
I would assume that these are always defined or undefined at the same time (you can't have a problem in a row without a column or without a sheet), so it would make sense to me to define a class `ExcelPosition` or `TableField` or something like that, that can bundle and serialize these three datapoints.